packages/php/email-editor/.wp-env.json

```diff
@@ -0,0 +1,30 @@
+{
+	"phpVersion": "7.4",
+	"plugins": [
+		"."
+	],
+	"config": {
+		"JETPACK_AUTOLOAD_DEV": true,
+		"WP_DEBUG_LOG": true,
+		"WP_DEBUG_DISPLAY": true,
+		"ALTERNATE_WP_CRON": true,
+		"WP_DEFAULT_THEME": "twentytwentyone"
+	},
+	"mappings": {
+	},
+	"env": {
+		"development": {
+			"phpmyadminPort": 9070
```

Comment:
Should we configure a different port or ports here? The default is 8888 and it conflicts when the Woo plugin wp-env is running. I assume we may want to run tests locally often, so it would be nice to be able to run it without the need to stop the Woo environment.

Reply:
Maybe we do not need phpmyadminPort in the default config because everybody can use an override file. Do you agree with it?

Reply:
I think we can keep it. I was thinking more about the mail port, which is 8888 by default, so currently, it conflicts with Woo env.